lib/cli_interface.rb

```diff
@@ -26,20 +24,34 @@ def set_lang(input)
     @text_file = YAML::load(File.open(file))
   end
 
-  def menu_game
-    write("#{get_from_file('menu_type_games')}")
+  def display_menu_type_game(default_checked_type = 1)
+    display_menu(get_from_file('menu_type_games'), default_checked_type)
   end
 
-  def menu_lang
-    display_lang_files
+  def menu_type_game_size
```

Comment:
the method is called 'menu_type_game_size', but the body takes 'menu_type_games' as an argument. 
This looks wrong no?  THe method name is talking about the board size, no?  But then it finds the number game types by counting the number of the lines in the file menu_type_games?  
These don't seem related to me.